README.md

```diff
@@ -123,6 +123,10 @@ Then, perform the following steps (on a clone/fork of Phing):
          $ cd test
          $ ../bin/phing
 
+Should this not work, you might be able to run the unit tests from the main directory:
+
+         $ $ ./bin/phing -f test/build.xml -Dtest=NotifyTaskTest
+
```

Comment:
@kenguest This is not needed. see #1086

Reply:
My bad - that was a straggler from a previous PR, removed it now.